src/ExadelMentorship.WebApi/Program.cs

```diff
@@ -7,6 +14,12 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddSingleton<CommandInvoker>();
```

Comment:
This registration is duplicated what in `DI` class. Try to find a way to introduce change that removes duplicate code.

Reply:
done